<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dangky_form</class>
 <widget class="QMainWindow" name="dangky_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng ký</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 243, 253)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);
border-radius: 10px
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>0</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Họ và tên</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTen">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>20</y>
       <width>196</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(233, 248, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>42</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ngày, tháng, năm sinh</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="txtNgaySinh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>196</width>
       <height>18</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>450</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(233, 248, 255)</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>85</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>105</y>
       <width>196</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(233, 248, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>125</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tên đăng nhập</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTenDangNhap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>145</y>
       <width>196</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(233, 248, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>165</y>
       <width>196</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mật khẩu </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMatKhau">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>185</y>
       <width>196</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(233, 248, 255)</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>205</y>
       <width>196</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nhập lại mật khẩu </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNhapLaiMatKhau">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>225</y>
       <width>196</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(233, 248, 255)</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDangKy">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>255</y>
       <width>68</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 219, 227)</string>
     </property>
     <property name="text">
      <string>Đăng ký</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnquaylai">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>275</y>
       <width>68</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 219, 227)</string>
     </property>
     <property name="text">
      <string>Đăng nhập</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHienMatKhau_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>185</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgba(245, 40, 145, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ic_hide.png</normaloff>../ic_hide.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAnMatKhau_2">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>225</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgba(245, 40, 145, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ic_visible.png</normaloff>../ic_visible.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHienMatKhau_4">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>225</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgba(245, 40, 145, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ic_hide.png</normaloff>../ic_hide.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>275</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bạn đã có tài khoản? </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAnMatKhau">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>185</y>
       <width>20</width>
       <height>18</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgba(245, 40, 145, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ic_visible.png</normaloff>../ic_visible.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>txtMatKhau</zorder>
    <zorder>label</zorder>
    <zorder>txtTen</zorder>
    <zorder>label_2</zorder>
    <zorder>txtNgaySinh</zorder>
    <zorder>label_3</zorder>
    <zorder>txtEmail</zorder>
    <zorder>label_4</zorder>
    <zorder>txtTenDangNhap</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>txtNhapLaiMatKhau</zorder>
    <zorder>btnDangKy</zorder>
    <zorder>btnquaylai</zorder>
    <zorder>btnAnMatKhau_2</zorder>
    <zorder>btnHienMatKhau_4</zorder>
    <zorder>label_7</zorder>
    <zorder>btnAnMatKhau</zorder>
    <zorder>btnHienMatKhau_3</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
